cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -148,7 +148,8 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
           |This will make sure no other constraint with the given name or another constraint of the same type and schema already exists before the specific `CreateConstraint` operator creates the constraint.
           |If it finds a constraint with the given name or with the same type and schema it will stop the execution and no new constraint is created.
           |The following query will create a unique constraint with the name `uniqueness` on the `name` property of nodes with the `Country` label only if
-          |no constraint named `uniqueness` or unique constraint on `(:Country {name})` already exists.""".stripMargin,
+          |no constraint named `uniqueness` or unique constraint on `(:Country {name})` already exists.
+          |Note: The `IF [NOT] EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards.""".stripMargin,
```

Comment:
Since this is `CREATE` the `NOT` should not be optional
```suggestion
          |Note: The `IF NOT EXISTS` syntax for constraints is only available in Neo4j 4.1.3 and onwards.""".stripMargin,
```